reframe/core/buildsystems.py

```diff
@@ -826,7 +826,8 @@ class Spack(BuildSystem):
     #: ReFrame looks for environments in the test's
     #: :attr:`~reframe.core.pipeline.RegressionTest.sourcesdir`.
     #:
-    #: This field is required.
+    #: If this field is `None`, the default, the environment name will
+    #: be automatically set to `rfm_spack_env`.
```

Comment:
I left the reference to `None` because it was already there, but I'm not sure that's accurate:
https://github.com/eth-cscs/reframe/blob/24321bb7e5a9f7a7e4e4486882d326c6809cad0f/reframe/core/buildsystems.py#L833
https://github.com/eth-cscs/reframe/blob/24321bb7e5a9f7a7e4e4486882d326c6809cad0f/reframe/core/buildsystems.py#L866

Reply:
Actually, this can be stale. I would change it to `None`. @rsarm does it ring a bell to you?

Reply:
This was changed in #2095